[2022-01-11 20:18:06,326] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_03 scheduled__2022-01-08T00:00:00+00:00 [queued]>
[2022-01-11 20:18:06,445] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_03 scheduled__2022-01-08T00:00:00+00:00 [queued]>
[2022-01-11 20:18:06,449] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:18:06,450] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-11 20:18:06,456] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:18:06,607] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_03> on 2022-01-08 00:00:00+00:00
[2022-01-11 20:18:06,629] {standard_task_runner.py:52} INFO - Started process 5398 to run task
[2022-01-11 20:18:06,659] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_03', 'scheduled__2022-01-08T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpbe78c7be', '--error-file', '/tmp/tmpxoiksds6']
[2022-01-11 20:18:06,678] {standard_task_runner.py:77} INFO - Job 44: Subtask task_03
[2022-01-11 20:18:11,540] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_03 scheduled__2022-01-08T00:00:00+00:00 [running]> on host airflowvm
[2022-01-11 20:18:12,436] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_03
AIRFLOW_CTX_EXECUTION_DATE=2022-01-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-08T00:00:00+00:00
[2022-01-11 20:18:12,463] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-11 20:18:12,476] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-01-11 20:18:12,547] {subprocess.py:85} INFO - Output:
[2022-01-11 20:18:15,643] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-11 20:18:15,801] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_03, execution_date=20220108T000000, start_date=20220111T201806, end_date=20220111T201815
[2022-01-11 20:18:15,969] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 20:18:16,098] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
